content/gemsets/using.haml

```diff
@@ -27,6 +27,13 @@ title: "'rvm gemset use' - Using gemsets"
     $ rvm gemset use teddy
 
 %p
+  To stop using the custom gemset (and switch to the default one) use one of the following:
+%pre.code
+  :preserve
+    $ rvm gemset use default
```

Comment:
when presenting two or more possibilities I usually add `# OR:` at the end of line